bitcash/network/services.py

```diff
@@ -27,20 +27,6 @@ class InsightAPI:
     MAIN_TX_AMOUNT_API = ''
     TX_PUSH_PARAM = ''
 
-    @classmethod
-    def get_balance(cls, address):
```

Comment:
Why are these being removed?

Reply:
Because they are not being used and are overwritten by BitcoreAPI.

Reply:
Gotcha!